src/pages/billing/List.js

```diff
@@ -44,8 +44,8 @@ const billingProjectNameValidator = existing => ({
   presence: { allowEmpty: false },
   length: { minimum: 6, maximum: 30 },
```

Comment:
this size limitation should also be removed

Reply:
I think we probably want to retain some size limitation for readability. I can check with Janki and UX to see if there's a product/design preference on this?

Reply:
I've reached out to Janki. I realized if we remove the size limitation, we also need to start dealing with text overflow, which means there's almost certainly design considerations.